### DISCUSSION TITLE
CPI Adjustment for 2022

### DISCUSSION THREAD

**Discussion by emehlhop** [USEPA/supply-chain-factors#57] [Q&A] [answered]
Hello, 
The provided code allows the user to select 2021 and prior year USD. I see in the description documents to use sector specific CPI to adjust for inflation, but I'm unsure where to find the sector specific values for 2021 and onwards in order to adjust. Does anyone have a recommendation? Currently I'm using average inflation to adjust for 2022 from https://www.bls.gov/data/inflation_calculator.htm
Thank you!

**Comment by bl-young** [comment, marked as answer]
[More recent data than 2021](https://apps.bea.gov/industry/Release/XLS/UGdpxInd/GrossOutput.xlsx) has not been published by BEA at the detail level, which is what is used by the model.

There are data at a higher level of resolution (which is more granular than just a single economy-wide inflation value) here:
https://apps.bea.gov/iTable/?reqid=150&step=2&isuri=1&categories=gdpxind

Select the `Chain-Type Price Indexes for Gross Output by Industry`

**Reply by emehlhop** [reply]
Thank you very much!
